### DISCUSSION TITLE
How to get this app configured - PLEASE add this to a top level readme.md

### DISCUSSION THREAD

**Discussion by TheDeuceYouSay** [Micke-K/IntuneManagement#279] [Q&A]
I poked around and went down multiple rat holes until I finally found instructions for getting this app configured.

Would you please add these instructions to the top level in the readme.md so it makes folks lives a little easier?

[Link to those instructions here.
](https://github.com/Micke-K/IntuneManagement/issues/229#issuecomment-2119086509)
Thank you !

**Comment by Micke-K** [comment]
Hello!

I'll add this to the documentation at the next release. 

Cheers!

**Comment by TheDeuceYouSay** [comment]
Thanks !

BTW – AMAZING tool you have built.  Thank you so much !
